File: 1421-find-numbers-with-even-number-of-digits/find-numbers-with-even-number-of-digits.cpp
class Solution {
public:
    int findNumbers(vector<int>& nums) {
        int result = 0;
        for (int num: nums){
            int dig_num = 0;
            while(num){
                num/=10;
                dig_num++;
            }
            if (dig_num%2==0){
                result++; 
            }
        
        }
        return result;
    }
};

File: 1421-find-numbers-with-even-number-of-digits/find-numbers-with-even-number-of-digits.py
class Solution:
    def findNumbers(self, nums: List[int]) -> int:
        n = len(nums)
        cnt = 0 
        for each in range(n):
            dig_len = len(str(nums[each]))
            if dig_len%2 == 0: 
                cnt += 1 
        return cnt